rails/benchmarks/activerecord_complex.rb

```diff
@@ -39,7 +41,7 @@ class Comment < ActiveRecord::Base
   end
 end
 
-m = Benchmark.measure do
+m = Benchmark.rails(100, "activerecord/#{ENV['DATABASE_URL'].split(":")[0]}/complex") do
```

Comment:
I mentioned it here: https://github.com/ruby-bench/ruby-bench-suite/pull/8#issuecomment-84039698

It can accept:

* `sqlite3::memory:`
* `postgres://username:password@localhost/myrailsdb`
* `mysql://user:password@localhost/ruby_bench`